source/pkcore/commands.rst

```diff
@@ -329,8 +329,7 @@ Who
 | Usage: ``/bending who [player]``
 | Aliases: ``[w]``
 
-Displays bending info about the given player, or lists all online
-players and their elements.
+Displays detailed bending info about the given player, or lists all online players and their elements.
```

Comment:
Change `given player` to `[player]`?

Reply:
I think given player is better because that syntax ("[Player]") should be reserved for commands.

Reply:
^ I didn't do that for that reason, and also because i didn't do that anywhere else, it would be slightly out of place.